src/site/content/en/identity/passkey-form-autofill/index.md

```diff
@@ -23,163 +23,264 @@ create a unified experience.
-
-1.  The user lands on the sign-in page.
-1.  The user selects the username input field.
-1.  An account selector dialog pops up.
-1.  The user selects an account, whether that account uses a password or a
-    passkey.
-1.  The user verifies with fingerprint, face or device PIN.
-1.  The user is signed in.
+To build an efficient user experience for both passkey and password users, you
+can include passkeys in autofill suggestions. This is called [conditional
+UI](https://github.com/w3c/webauthn/wiki/Explainer:-WebAuthn-Conditional-UI) and
+it's a part of [the WebAuthn standard](https://w3c.github.io/webauthn/).
+
+<figure>
+  {%
+    Video src="video/YLflGBAPWecgtKJLqCJHSzHqe2J2/ZaxgMtkOh5CX2B1ZT9gd.mp4",
+    autoplay="true", loop="true"
+  %}
+</figure>
 
 As soon as the user taps on the username input field, an autofill suggestion
 dialog pops up which highlights the stored passkeys along with password autofill
 suggestions. The user can then choose an account and use the device screen lock
 to sign in.
 
-This way, users can sign in to your website with the existing form as if
-nothing has changed, but with
-[the added security benefit of passkeys](https://developers.google.com/identity/passkeys#security-considerations)
+This way, users can sign in to your website with the existing form as if nothing
+has changed, but with [the added security benefit of
+passkeys](https://developers.google.com/identity/passkeys#security-considerations)
 if they have one.
 
-{% Aside %}
+{% Aside 'success' %}
 
-When a user signs in with a password, consider using that opportunity to
-offer them to
-[create a passkey](/passkey-registration).
+When a user signs in with a password, consider using that opportunity to offer
+them to [create a passkey](/passkey-registration).
 
 {% endAside %}
 
+## How it works
+
+To authenticate with a passkey, you use [the WebAuthn 
+API](https://w3c.github.io/webauthn/).
+
+In a passkey authentication flow, four participants are involved in addition to 
+the user:
+
+* **Backend**: Your accounts database or backend server that stores the public 
+  key and other metadata about the passkey.
+* **Frontend**: Your frontend server which communicates with it and serves 
+  requests.
+* **Browser**: The user's browser which is running your Javascript.
+* **Authenticator**: The user's authenticator which creates and stores the 
+  passkey. This may be on the same device as the browser (e.g. when using 
+  Windows Hello) or on another device, like a phone.
+
+<figure class="screenshot">
+  {%
+    Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/vAwnMiLi7U1spDopYg2Q.png",
+    alt="Passkey authentication diagram", width="800", height="254"
+  %}
+</figure>
+
+1. As soon as a user lands on the frontend, it requests a challenge from the 
+   backend to authenticate with a passkey and calls 
+   `navigator.credentials.get()` to initiate authenticating with a passkey. 
+   This returns a promise.
```

Comment:
Do we define what a "promise" is anywhere?

Reply:
```suggestion
   This returns a `Promise`.
```